org.adempiere.base/src/org/compiere/util/WebEnv.java

```diff
@@ -186,15 +188,55 @@ public static boolean initWeb (ServletContext context)
 		//	Logging now initiated
 		if (log.isLoggable(Level.INFO)) log.info(info.toString());
 		//		
-		MClient client = MClient.get(Env.getCtx(), 0);
-		MSystem system = MSystem.get(Env.getCtx());
-		client.sendEMail(client.getRequestEMail(),
-			"Server started: " + system.getName() + " (" + WebUtil.getServerName() + ")",
-			"ServerInfo: " + context.getServerInfo(), null);
+
+		serverStartEMail(context);
 
 		return s_initOK;
 	}	//	initWeb
 
+	/*
+	 * Send an email when the server starts
+	 * @param context servlet context
+	 */
+	private static void serverStartEMail(ServletContext context) {
+
+		boolean mailSent = false;
+
+		MClient client = MClient.get(Env.getCtx(), 0);
+		MSystem system = MSystem.get(Env.getCtx());
+
+		String recipient = MSysConfig.getValue("EMAIL_SERVER_START_RECIPIENT", 0, 0);
+		if (Util.isEmpty(recipient) || !EMail.validate(recipient))
+			recipient = client.getRequestEMail();
+
+		int mailtextID = MSysConfig.getIntValue("EMAIL_SERVER_START_MAILTEXT_ID", 0, 0);
+		if (mailtextID > 0) {
+
+			try {
+				MMailText mt = new MMailText(Env.getCtx(), mailtextID, null);
+				mt.setPO(client);
+				String subject = mt.getMailHeader();
+				subject = subject.replace("#SYSTEM_NAME#", system.getName());
+				subject = subject.replace("#SERVER_NAME#", WebUtil.getServerName());
+
+				String message = mt.getMailText(true);
+				message = message.replace("#SERVER_INFO#", context.getServerInfo());
+				message = message.replace("#ADEMPIERE_VERSION#", Adempiere.getVersion());

```

Comment:
Instead using a new convention, I think it is better to use back the current mail template convention for variable (i.e `@variableName@`) and use back the existing mail template parse function. For variables that's not coming from MSystem and MClient, you can create a new Java class for that (i.e the plain Java bean class should have `getServerName`, `getServerInfo` and `getVersion` methods).